src/core_plugins/metrics/public/visualizations/components/timeseries.js

```diff
@@ -68,7 +68,7 @@ class Timeseries extends Component {
               if (row.data[i] && pos.x < row.data[i][0]) break;
             }
             if (!row.data[closest]) return values[row.id] = null;
-            const [ time, value ] = row.data[closest];
+            const [ , value ] = row.data[closest];
```

Comment:
eslint complains, and I think that's the right thing to do unless we want to pick a variable pattern we can use for unused placeholders (something we can tell eslint to ignore, like `unused{N}`)